articles/overview/what-is-qsharp-and-qdk.md

```diff
@@ -0,0 +1,66 @@
+---
+title: What is the Q# programming language and QDK?
+description: Learn about the Microsoft Quantum Development Kit, the Q# programming language, and how you can create quantum programs.
+author: bradben
+ms.author:  bradben
+ms.date: 04/22/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.q-sharp
+---
+
+# What is the Q# programming language and QDK?
+
+Q# is Microsoft’s open-source programming language for developing and running quantum algorithms. It’s part of the Quantum Development Kit (QDK), which includes Q# libraries, quantum simulators, extensions for other programming environments, and API documentation. In addition the Standard Q# library, the QDK includes Chemistry, Machine Learning, and Numeric libraries.
```

Comment:
Given this PR targets `master`, it would be good for these to be links to other parts of the docs; if the concept guide needs further work before publication (e.g.: aforementioned links), then I'd strongly suggest targeting a feature branch with this PR instead of master.